.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,11 @@
+{
+	"name": "Python 3",
+	"image": "mcr.microsoft.com/devcontainers/python:1-3.10-bookworm",
+	"features": {
+		"ghcr.io/devcontainers/features/docker-in-docker:2": {}
+	},
+	"postStartCommand": "bash .devcontainer/setup_odbc.sh && bash .devcontainer/install_pyenv.sh && bash .devcontainer/setup_env.sh",
+	"runArgs": [
+        "--env-file", "${localWorkspaceFolder}/.devcontainer/test.env"
```

Comment:
Is the the correct path to the `test.env` file? The contributing guide states to place it in the root directory.

Reply:
But also on the topic of this merge, independently verified passing tests would be superb if we need to go around this weird CI system we got goin on

Reply:
@schlich my env is up and I'm close on getting the tests updated so they pass. I have some time set aside this evening to finish that up (7-9PM MST). I'd like to finish that unless you plan to do it.

Reply:
Awesome!  Yeah, I was running into some env-related bugs last night so definitely feel free to run with this